### ISSUE TITLE
new net no.260 promoted too early

### ISSUE THREAD

**Issue by l1t1** [CuriosAI/sai#103]
through the net 8743e8d3 performance very nice vs lz nets
it's winrate vs net no.259 is only 0.583, there are 3 nets have higher winrate 
![2020-01-26_203643](https://user-images.githubusercontent.com/23301704/73135431-c563dd00-407c-11ea-8979-fe553120d65f.png)

**Comment by l1t1** [comment]
when i was writing the post, you tested 5bb4703d too :)

**Comment by trinetra75** [comment]
Yes, we added both nets for comparison matches (as it is a manual process), but only 5bb4 will be promoted.
Promotion happens at the end of the self-play quota, so that we are able to add extra training runs in case [as happened this time].

**Comment by l1t1** [comment]
The promoted net 5bb4703d isnt in the list yet, but the next round matches are running.
![2020-01-27_175146](https://user-images.githubusercontent.com/23301704/73164984-f18b6680-412d-11ea-8041-3f97604fa649.png)

**Comment by l1t1** [comment]
the list is ok now

**Comment by trinetra75** [comment]
The promoted net appears in the list on the left only after the first self-play has finished.
In the last few days we moved back all clients to run the matches in case any are running (games per hours is a bit on the low side...), so self-plays might start quite late!

**Comment by barrtgt** [comment]
Net abc7474f has yet to do self play games, yet it is looking for a successor.
